.changeset/new-rats-return.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I believe the changeset should not be empty since we are adding a new exportable helper on the `crypto` package and made changes on the `wallet` package. 

Therefore we want to bump versions on these 2 packages.

Reply:
These packages are published independently.

We can't consider things as "internal" in this way.

The usual is to bump everything that will get a new published version.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1608/commits/732d8f9ff2d86e42aa0de35db519e9eb258e01a2